R/deprecated.R

```diff
@@ -0,0 +1,118 @@
+#' Forward geocoding
```

Comment:
should there be "more deprecation" i.e. e.g. a message even when using a single placename?

Reply:
Yes, still need to do that and also deprecate `opencage_key` (and not completely delete it). I've started with the latter [in this branch](https://github.com/dpprdan/opencage/tree/fix_opencage_key) and would do the former there as well.